lib/matplotlib/tests/test_mathtext.py

```diff
@@ -12,7 +12,7 @@
 from matplotlib import mathtext
 
 math_tests = [
-    r'$a+b+\dots+\dot{s}+\ldots$',
+    r'$a+b+\dot s+\dot{s}+\ldots$',
```

Comment:
\dots is a macro in amsmath, it chooses a suitable height for the dots: http://tex.stackexchange.com/questions/649/how-do-magic-dots-work-in-amsmath